### PR TITLE
[FLINK-23484] Supports to test ChangelogStateBackend

### PR DESCRIPTION
**Pull request by masteryhx**
See [FLINK-23484](https://issues.apache.org/jira/browse/FLINK-23484) for more details.

### CODE REVIEW

File: src/main/java/org/apache/flink/state/benchmark/StateBenchmarkBase.java [hunk 1]
```diff
@@ -57,7 +56,7 @@ public class StateBenchmarkBase extends BenchmarkBase {
     static AtomicInteger keyIndex;
     final ThreadLocalRandom random = ThreadLocalRandom.current();
 
-    @Param({"HEAP", "ROCKSDB"})
+    @Param({"HEAP", "ROCKSDB", "HEAP_CHANGELOG", "ROCKSDB_CHANGELOG"})
```
Comment (pnowojski) [inline src/main/java/org/apache/flink/state/benchmark/StateBenchmarkBase.java:59]: I would remove `HEAP_CHANGELOG` from the runs. This sounds like a quite unlikely production combination.

Comment (masteryhx) [reply src/main/java/org/apache/flink/state/benchmark/StateBenchmarkBase.java]: Thanks for the advice.
I aggree that we should focus on testing the production env.

**Comment by Myasuka** [comment]
@masteryhx please checked that why the CI failed.

**Comment by masteryhx** [comment]
@Myasuka 
I saw it's exising exception which may occur accidentally, see previous workflow [#74](https://github.com/apache/flink-benchmarks/actions/runs/5219158886/jobs/9450453549#logs), [#40](https://github.com/apache/flink-benchmarks/actions/runs/4915916989/jobs/8779014239).
I just reruned this to see whether it could be green.
If it's true, maybe it deserves a seprate ticket to make the CI stable.

**Comment by Myasuka** [comment]
@masteryhx could you get a green run in your forked CI?

**Comment by masteryhx** [comment]
> @masteryhx could you get a green run in your forked CI?

Yeah, I got a green in my forked one: https://github.com/masteryhx/flink-benchmarks/actions/runs/5572457248

**Review by Myasuka — APPROVED** [review]
LGTM, please create a separate ticket for the failed CI.

**Pull request merged**